Simplified/AxisService/NYPLAxisService.swift

```diff
@@ -378,34 +180,67 @@ extension NYPLAxisService {
         return nil
     }
     
+    self.init(axisKeysProvider: axisKeysProvider,
+              book: book,
+              bookVaultId: bookVaultId,
+              cypher: cypher,
+              delegate: delegate,
+              deviceInfoProviding: deviceInfoProviding,
+              fileURL: fileURL,
+              isbn: isbn)
+  }
+  
+  /// Convenience initialzer to be used for testing
+  /// - Parameters:
+  ///   - axisKeysProvider: NYPLAxisKeysProviding object
+  ///   - book: NYPLBook that is to be downloaded
+  ///   - bookVaultId: BookVaultID for the given book
+  ///   - cypher: NYPLRSACryptographing object to be used for downloading license
+  ///   - delegate: An object confirming to NYPLBookDownloadBroadcasting protocol
```

Comment:
minor: instead of repeating the declaration from the code, can you say why we need this delegate or what are we delegating?

Generally speaking, if the docs don't provide anything additional to the code (because the meaning is obvious and there's nothing to clarify), it's fine to omit them. We don't `have to` put something in just because.

another example: I can tell `bookVaultId` is the vaultID for the given book by just reading the declaration, but what's less obvious is the meaning of "vault" in this context. IMO that's the kind of stuff that makes great docs useful. I like what you did for `cypher` and `book` for example.

Reply:
Done